packages/core/src/types.ts

```diff
@@ -1275,18 +1275,25 @@ export interface ActorRef<TEvent extends EventObject, TEmitted = any>
   send: Sender<TEvent>;
 }
 
-export interface SpawnedActorRef<TEvent extends EventObject, TEmitted = any>
-  extends ActorRef<TEvent, TEmitted> {
+export interface SpawnedActorRef<
```

Comment:
writing down somewhere what is considered to be an Actor, ActorRef and SpawnedActorRef would be very helpful - I often have to recheck this on my own in the codebase as it's not entirely intuitive for me (especially the distinction between `ActorRef` and `SpawnedActorRef`

Reply:
Could write that down right here to start 😁 Pretty please!

Reply:
Added as todo here: https://github.com/davidkpiano/xstate/projects/1#card-61626855